quantlib/test/test_date.py

```diff
@@ -61,7 +61,7 @@ def test_comparison_with_datetime(self):
         datetime_date_nov_98 = datetime.date(1998, 11, 19)
 
         self.assertTrue(date_nov_98 == datetime_date_nov_98)
-        self.assertEquals(cmp(date_nov_98, datetime_date_nov_98), 0)
+        self.assertEquals(date_nov_98 > datetime_date_nov_98, 0)
```

Comment:
Not for this one but for the next test sure. This test is about testing the internal comparison methods. The result should not be an int but a boolean. I'm fixing that